setup.py

```diff
@@ -22,8 +22,8 @@
     "cachetools>=2.0.0,<5.0",
     "pyasn1-modules>=0.2.1",
     # rsa 4.1, 4.1.1, 4.2 are broken on Py2: https://github.com/sybrenstuvel/python-rsa/issues/152
-    'rsa>=3.1.4,!=4.1,!=4.1.1,!=4.2,<5; python_version < 3',
-    'rsa>=3.1.4,<5; python_version >= 3'
+    "rsa>=3.1.4,!=4.1,!=4.1.1,!=4.2,<5; python_version < 3",
```

Comment:
I think "3" may need to be in quotes as in:
`'rsa>=3.1.4,!=4.1,!=4.1.1,!=4.2,<5; python_version<"3"'`
`'rsa>=3.1.4,<5; python_version>="3"'`

Reply:
Currently fixing and testing locally

Reply:
My motivation for rsa>=3.1.4,!=4.1,!=4.1.1,!=4.2  was that there maybe another Py2 release (which we had) with additional fixes (there were security backports), and the range would be able to pick it up.
I also was hoping that new releases won't be broken on Py2, however, looks like 4.4 is installable and broken on Py2, so we'd have to exclude it.

Reply:
Narrowed in in #532 

https://github.com/googleapis/google-auth-library-python/blob/b74168b710da38c0cf06cdf8685c49af4748e3a0/setup.py#L25-L26

Reply:
Released `1.17.2` with the pin above